.github/workflows/populate-gradle-build-cache.yml

```diff
@@ -0,0 +1,28 @@
+# Build the app on each push to `develop`, populating the build cache to speed
+# up CI on PRs.
+
+name: Populate build cache
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
What's the difference to ci.yml?
Does that one only build on tag pushes?

Reply:
ci.yml runs on every push. This one will run only when stuff is pushed to develop (which means, when a PR is merged to develop).

Reply:
And if ci.yml runs on every push (also merges to develop) does this populate then do something more?

Reply:
Sorry, should have been clearer. ci.yml runs when a pull request is opened (`pull_request:`), when a pull request has a commit pushed to it, or manually (`workflow_dispatch`).

It doesn't trigger on pushes to `develop`. That's what this one is for.

It's separate because there's no need to run the lint/test steps, because the PR will need to have passed those before it can be merged.